nntrainer/layers/layer_context.h

```diff
@@ -513,10 +514,18 @@ class RunLayerContext {
    * @brief check current graident is first access
    *
    * @param idx index
-   * @return bool true if last access
+   * @return bool true if first access
    */
   bool isGradientFirstAccess(unsigned int idx) const;
 
+  /**
+   * @brief check current graident is last access
+   *
+   * @param idx index
+   * @return bool true if last access
```

Comment:
add a note - equivalent to weight first access

Reply:
Done